### PR TITLE
Adding projects to the list.

### PR DESCRIPTION
**Pull request by ttoine**
work in progress.

### CODE REVIEW

File: coreteam.py [hunk 1]
```diff
@@ -61,19 +61,39 @@
...
     'cronjobs': 'Cronjobs module',
     'dashactivity': 'Dashboard Activity module',
+    'dashgoals': 'Dashboard Goals',
+    'dashproducts': 'Dashboard Products',
+    'dashtrends': 'Dashboard Trends',
     'example-modules': 'Example modules',
     'gamification': 'Gamification module',
+    'gridhtml': 'Simple HTML table display',
     'gsitemap': 'Google Sitemap module',
+    'graphnvd3'; 'NVD3 Charts',
```
Comment (ttoine) [inline coreteam.py]:
```suggestion
    'graphnvd3': 'NVD3 Charts',
```

**Comment by ttoine** [comment]
I tested this pull request locally and it works, but as I added a looot of new projects to the list I would appreciate a review and a test from someone else.

File: coreteam.py [hunk 2]
```diff
@@ -82,12 +102,34 @@
...
+    'statsnewsletter': 'Newsletter',
+    'statsorigin': 'Visitors origin',
+    'statspersonalinfos': 'Registered customer information',
+    'statsproduct': 'Product details',
     'statsregistrations': 'Registrations statistics modules',
+    'statssales': 'Sales and orders',
+    'statssearch': 'Shop search',
+    'statsstock': 'Available quantities',
+    'statsvisits': 'Visits and Visitors',
+    'welcome': 'OnBoarding',
```
Comment (Progi1984) [inline coreteam.py:135]: Please precise these are stats

Comment (ttoine) [reply coreteam.py:135]: I actually used the module names in their repository's readme.md files. So, should I also update all the correspondant module names in each repository? What do you think?

Comment (Progi1984) [reply coreteam.py:135]: @PrestaShop/prestashop-product-team The name of project doesn't contain "Statistics" or "Stats". Is it wanted ?

Comment (sam-pires) [reply coreteam.py:135]: what is this about? 😅

Comment (Progi1984) [reply coreteam.py:135]: @sam-pires Module name for "stats*" doesn't indicate that there are about statistics.

**Comment by ttoine** [comment]
can we merge that anyway, if there is no update of the name of the projects ?

File: coreteam.py (hunk 2 above)
Comment (Progi1984) [reply coreteam.py:135]: @sam-pires No answer to my question ?

Comment (sam-pires) [reply coreteam.py:135]: Hello 🙂 
Do we just need to change the names in this file or to update them everywhere the modules' names are mentioned?

If it is just here, we can add "xxx stats" at the end of their names
If it is everywhere, let's keep it that way for the moment

**Comment by PierreRambaud** [comment]
> can we merge that anyway, if there is no update of the name of the projects ?

https://github.com/PrestaShop/core-weekly-generator/pull/29#discussion_r410061164 comments have been added :wink:

**Comment by sam-pires** [comment]
How urgent is it @ttoine?

**Comment by ttoine** [comment]
I use it every week to generate the core weekly. Until it's not merged, I will use the list in this current PR (via my local git clone), it saves a lot of time instead of having to edit manually the names.

@sam-pires if you feel this is not urgent to change the name of the modules in their repository, we can stay with this. Just, when a module name is improved in its repository with "stats" like @Progi1984 suggested, we will have to think about updating the list in this file.

**Comment by sam-pires** [comment]
Ok let's merge then.
if it's a pain in the ass for the reader, they will remind us to update the names.

**Comment by ttoine** [comment]
Can someone merge this then, please ??

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
@ttoine After discussion with @sam-pires, an issue will be created for renaming these modules. I merge your PR.

**Comment by sam-pires** [comment]
FYI here is the issue  https://github.com/PrestaShop/PrestaShop/issues/18738

**Comment by ttoine** [comment]
Thanks folks :-)

**Pull request merged**